informer/src/handler/rpc_ws_handler.rs

```diff
@@ -87,15 +91,17 @@ impl Handler {
                     }
                 };
                 let mut rng = rand::thread_rng();
-                let sub_id = rng.gen();
+                let sub_id = rng.gen_range(0, 1_000_000_000);
```

Comment:
Why do you use `rng.gen_range(0, 1_000_000_000)`?
It will make the random value more likely to conflict.